crds/lvmvolumegroupset.yaml

```
@@ -166,15 +181,15 @@ spec:
                             x-kubernetes-int-or-string: true
                             pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$|^[1-9][0-9]?%$|100%'
                             description: |
-                              The desired thin pool size. Might be specified as number or percent size of total VG space.
+                              The desired thin pool size. Might be specified as a number or percentage of total VG space.
                               
-                              > Note, that if you specify the percent size, the thin pool will be automatically extended when VG is extended.
+                              > Note that if you specify the percent size, the thin pool will be automatically extended when VG is extended.
                           allocationLimit:
                             type: string
                             pattern: '^[1-9][0-9]{2,3}%$'
                             default: "150%"
                             description: |
-                              Thin pool oversize limit. Default is 150%.
+                              Thin pool oversize limit. Default: 150%.
```

```suggestion
                              Thin pool oversize limit.
```
The default value is already described in the `default` field.